frontend/awx/views/schedules/hooks/useSchedulesFilter.tsx

```diff
@@ -0,0 +1,17 @@
+import { useMemo } from 'react';
+import { IToolbarFilter } from '../../../../../framework';
+import {
+  useDescriptionToolbarFilter,
+  useNameToolbarFilter,
+} from '../../../common/awx-toolbar-filters';
+
+export function useSchedulesFilter() {
+  const nameToolbarFilter = useNameToolbarFilter();
+  const descriptionToolbarFilter = useDescriptionToolbarFilter();
+
+  const toolbarFilters = useMemo<IToolbarFilter[]>(
+    () => [nameToolbarFilter, descriptionToolbarFilter],
+    [nameToolbarFilter, descriptionToolbarFilter]
+  );
+  return toolbarFilters;
```

Comment:
the awx implementation includes filters for `Created By` and `Modified By`. It would be good to have those too.